Dockerfile

```diff
@@ -1,13 +1,13 @@
 # stage 1 Generate celestia-appd Binary
 FROM golang:1.18-alpine as builder
-RUN apk update && apk --no-cache add make gcc musl-dev
+RUN apk update && apk --no-cache add gcc=latest musl-dev=latest
```

Comment:
[Non Blocking][Question]
> By default apk tries to select a set of packages so that all dependencies are satisfied. This means that some packages can be held back in an older version if there is direct or implicit dependencies to the older version.
https://man.archlinux.org/man/apk-add.8.en

I guess it's better not to force `latest` as it might not be compatible with the already installed packages in the base image. What do you think?

Reply:
So these were `WARNINGS` about not specifying a version. In general it is best practice to use specific versions but `apk` is a nice tool.

If we merge this https://github.com/celestiaorg/.github/pull/8 I can revert these changes.

Reply:
Understandable. I am fine with whatever you prefer :cat:

Reply:
Went with ignoring this rule inline in favor of `apk` compatibility.